app/main.py

```diff
@@ -1,18 +1,38 @@
-from fastapi import FastAPI, Request
-from fastapi.staticfiles import StaticFiles
+from typing import Optional
+from datetime import datetime
+
+
+from fastapi import Depends, FastAPI, Request, Form
+from fastapi.responses import HTMLResponse
+# from fastapi.staticfiles import StaticFiles
 from fastapi.templating import Jinja2Templates
 
+from app.database import models
+from app.database.models import Event
+from app.database.database import SessionLocal, engine
+
+models.Base.metadata.create_all(bind=engine)
+
+
+
+def get_db():
```

Comment:
Try not to add content to main.py. Also, this line can be found on `dependencies.py`